### PR TITLE
Drinking Space Lube Makes Preternis Move a Bit Faster

### PR DESCRIPTION
**Pull request by UselessTheremin**
### Intent of your Pull Request

Gives lube the ability to make preternis go a bit faster. Just giving the chemical a bit more interaction and possible use for preternis, nothing huge

### Why is this change good for the game?

Small little features that make you go "oh neat" are good in my mind. Also, lube make motor go fast, right?? 🧠 

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 

Gives Space Lube the trait of making preternis move faster.

### What general grouping does this PR fall under? 

Chemistry and Preternis 

### If there are any numerical values involved in your PR that will be relevant to a player, please note them here. 

The lube makes preternis go 1.25x as fast as normal

:cl:  
tweak: preternis now gain a small speed boost when lube is in their system
/:cl:

tell me if the code doesnt work i dont have time to test it today 😔

### CODE REVIEW

**Comment by Mqiib** [comment]
Ah yes speed boosts with no downsides, we need more of those.

**Comment by UselessTheremin** [comment]
you know what yeah this is retarded
